src/components/Reports/GDPRPermissions/PermissionExpandableRow.js

```diff
@@ -0,0 +1,175 @@
+import React, { useState, useEffect } from 'react';
+import * as S from './SubscriberGdpr.styles';
+import { FormattedMessage, useIntl, FormattedDate } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Loading } from '../../Loading/Loading';
+import { InjectAppServices } from '../../../services/pure-di';
+
+const PermissionValue = ({ value }) => {
+  const permissionValue = value.toLowerCase();
+  let iconColor =
```

Comment:
Why do you use `let` here? You are not changing this iconColor variable... 🤔 
```suggestion
  const iconColor =
```